hc/accounts/tests/test_profile.py

```diff
@@ -35,8 +35,8 @@ def test_it_sends_report(self):
         self.alice.profile.send_report()
 
         # Assert that the email was sent and check email content
-        self.assertIn("This is a monthly report sent by healthchecks.io.", \
-                      mail.outbox[-1].body)
+        self.assertIn("Healthchecks Report", \
```

Comment:
Split this test into daily, weekly and monthly. Check the email content for the appropriate message